app/templates/admin/manageDepartments.html

```diff
@@ -62,4 +63,43 @@ <h1>Manage Departments</h1>
         <div class="col-md-2"></div>
     </div>
 </div>
+<div class="modal" id="manageDepartmentSupervisorModal" tabindex="-1" role="dialogue" >
+  <div class="modal-dialog" role="document">
+    
+    
+    <div id="manageSupervisorContent" class="modal-content">
+      
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        <div class="department-name"></div>
+        
+      </div>
+
+      <div class="modal-body ">
+        <div class="form-group">
+        <label for="supervisorModalSelect">Select a supervisor to add:</label>
+        <div class="dropdown"><select class="selectpicker"
+          name='supervisor'
+          id='supervisorModalSelect'
+          data-live-search='true'
+          title="Supervisor">
+        {% for supervisor in allSupervisors  %}
+          <option value="{{supervisor.ID}}"
+                  data-content="{{supervisor.FIRST_NAME}} {{supervisor.LAST_NAME}}<small class='text-muted'> ({{supervisor.ID}})</small>">
+            {{supervisor.FIRST_NAME}} {{supervisor.LAST_NAME}}
+          </option>
+        {% endfor %}
+      </select></div>
+      </div>
+        <div class="changing-content"></div>
+      </div>
+      <div class="modal-footer">
+        <button type="button" id="addSupervisorsToDepartmentSubmit" data-dismiss="modal" class="btn btn-primary">Done</button>
```

Comment:
Correct me if I'm wrong, but I think that the supervisors are actually added whenever they're accessed via the select and then added to our DB. Therefore, calling this button's id `addSupervisorsToDepartmentSubmit` might be misleading. Consider renaming it to something like `exit`.